src/router/apiRouter.js

```diff
@@ -33,4 +34,33 @@ router.post("/login", async (req, res) => {
   }
 });
 
+router.post("/authentication/login", async (req, res) => {
+  const { username, password } = req.body;
+
+  const setCookie = (userInformation) => {
+    res.cookie('user', JSON.stringify(userInformation), { 
+      expires: new Date(Date.now() + 7 * 24 * 3600 * 1000),
+      httpOnly: true
+    });
+  };
+
+  usersRepository.findUserByUsernameAndPassword(username, password)
```

Comment:
@gracegately 
In this example, it will redirected to `my-account` even if the promise resolved or rejected.
Is it the correct behaviour?

Reply:
Yes -- I think this can be improved, though. For now, even though the user goes to `/my-account`, that route will redirect them to authentication if there's no user on the cookie.